.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,26 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build 
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
```

Comment:
Can we change it to the latest stable, meaning 12?

Reply:
I was getting build issues with anything other than 10 :(

I think we should tackle a node environment update separately from CICD effort

Reply:
Sure, no problem